explore/src/main/scala/explore/Explore.scala

```diff
@@ -172,8 +170,11 @@ object ExploreMain {
         _                    <- setupReusabilityOverlay(appConfig.environment)
         r                    <- (ctx.sso.whoami, setupDOM[IO], showEnvironment[IO](appConfig.environment)).parTupled
         (vault, container, _) = r
-      } yield RootComponent(ctx, router, initialModel(vault, localPreferences))
-        .renderIntoDOM(container)
+      } yield ReactDOMClient
+        .createRoot(container)
+        .render:
+          RootComponent(ctx, router, initialModel(vault, localPreferences)).toUnmounted
```

Comment:
I don't really understand what appears to be a type ascription to `RootComponent(ctx...`

Reply:
You mean the `toUnmounted`? That has to do with new `Renderable` typeclass implemented in scalajs-react. I think the long-term strategy to avoid this is to create an instance of the typeclass for classes that extend `ReactProps`/`ReactFnProps`/etc.

Reply:
I was thinking of the `.render: RootComponent(...`

Reply:
It's not a type annotation, it's the braceless syntax for functions

```scala
x.map: y =>
  y + 1
```